### ISSUE TITLE
Questions about LDM and SD-VAE

### ISSUE THREAD

**Issue by Lucien66** [hustvl/LightningDiT#9]
Thank you for your great work！ However, I am slightly confused about the relationship between LDM in Table 2, SD-VAE and LDM* in Table 3.
1. The LDM in Table 2 and the SD-VAE in Table 3 have the same references, can I assume that they are equivalent?
2. May I consider LDN+VF loss = VA-VAE? or SD-VAE+VF loss = VA-VAE?
3. What is the difference between LDM in Table 2 and LDM*, used in MAR?
I would be most grateful if you could make some clarifications.
Thank you!!

**Comment by JingfengYao** [comment]
Your observation is very meticulous.
1. Not quite. This is because SD-VAE was trained on the internal dataset. When we refer to LDM in Tables 2 and 3, we primarily point to its model architecture.
2. In our paper, we employ the LDM autoencoder baseline to validate the effectiveness of the VF loss. However, the VF loss and LDM should be decoupled, allowing the VF loss to function as a plug-and-play module that can be integrated into more VAE architectures.
3. A slight difference is that MAR does not utilize the attention layer from the LDM autoencoder. Moreover, our training details are not entirely identical.

**Comment by Lucien66** [comment]
Thank you very much for your quick and clear reply！
Again, Great work!! Thanks a lot!!
